apps/fulfillment-e2e/src/integration/start-picking-picklist-with-customer-note.cy.ts

```diff
@@ -0,0 +1,122 @@
+import { CustomerNoteFragment } from '../support/page_fragments/customer-note.fragment';
+import { PickingListsFragment } from '../support/page_fragments/picking-lists.fragment';
+import { PickingFragment } from '../support/page_fragments/picking.fragment';
+
+const pickingListsFragment = new PickingListsFragment();
+const customerNoteFragment = new CustomerNoteFragment();
+
+describe('Start picking a picklist with customer note', () => {
+  let identifier = '';
+
+  beforeEach(() => {
+    cy.clearIndexedDB();
+    cy.visit('/');
+    pickingListsFragment
+      .getCustomerNoteButton()
+      .parents('oryx-picking-list-item')
+      .within(() => {
+        pickingListsFragment
+          .getIdentifier(true)
+          .invoke('text')
+          .then((text) => {
+            identifier = text.trim();
+
+            pickingListsFragment.getStartPickingButton(true).click();
+          });
+      });
+  });
+
+  it('should navigate to correct url', () => {
+    cy.url().should('include', `/customer-note-info/${identifier}`);
+  });
+
+  it('should display customer note', () => {
+    customerNoteFragment
+      .getNote()
+      .should('be.visible')
+      .and('contain', 'Some cart note');
+  });
```

Comment:
Atomic tests are very cool and I like that)
but, we use Cypress Dashboard and pay for EACH test. 

That's why we should combine checks it 1-2 tests